packages/office-addin-usage-data/src/usageData.ts

```diff
@@ -108,15 +108,14 @@ export class OfficeAddinUsageData {
   public async reportEventApplicationInsights(eventName: string, data: object): Promise<void> {
     if (this.getUsageDataLevel() === UsageDataLevel.on) {
       const usageDataEvent = new appInsights.Contracts.EventData();
-      usageDataEvent.name = eventName;
+      usageDataEvent.name = `${eventName}${this.options.isForTesting ? "-test" : ""}`;
       try {
         for (const [key, [value, elapsedTime]] of Object.entries(data)) {
-          usageDataEvent.properties[key] = value;
+          usageDataEvent.properties[key] = `${value}${this.options.isForTesting ? "-test" : ""}`;
```

Comment:
I'm confused about this. Shouldn't this be adding ` "isForTesting" = isForTesting ` after the passed in values in data?